calico-enterprise/getting-started/bare-metal/about.mdx

```diff
@@ -40,94 +40,108 @@ To learn how to restrict traffic to/from hosts and VMs using Calico network poli
 
 ## How to
 
-### Set up your Kubernetes cluster to work with a non-cluster host or VM
-
-1. Apply the `NonClusterHost` custom resource. The `endpoint` field is required. It defines the remote endpoint for the non-cluster host log forwarder.
-
-   ```bash
-   kubectl create -f - <<EOF
-   apiVersion: operator.tigera.io/v1
-   kind: NonClusterHost
-   metadata:
-     name: tigera-secure
-   spec:
-     endpoint: <endpoint for non-cluster host log forwarder>
-   EOF
-   ```
-
-   Wait until the manager shows a status of Available, then proceed to the next step.
-
-1. Obtain the token for `tigera-noncluster-host` service account.
-
-   ```bash
-   kubectl get secret -n calico-system tigera-noncluster-host -o jsonpath='{.data.token}' | base64 --decode
-   ```
-
-1. Use a text editor to create a kubeconfig file for non-cluster host or VM.
-
-   ```yaml
-   apiVersion: v1
-   kind: Config
-   current-context: noncluster-hosts
-   preferences: {}
-   clusters:
-   - cluster:
-       certificate-authority-data: <your cluster certificate>
-       server: <your cluster server>
-     name: noncluster-hosts
-   contexts:
-   - context:
-       cluster: noncluster-hosts
-       user: tigera-noncluster-host
-     name: noncluster-hosts
-   users:
-   - name: tigera-noncluster-host
-     user:
-       token: <token from previous step>
-   ```
-
-   Take the cluster information from an already existing kubeconfig.
-
-1. Create a [`HostEndpoint` resource](../../reference/host-endpoints/overview.mdx) for each non-cluster host or VM. The `node` and `expectedIPs` fields are required to match the hostname and the expected interface IP addresses.
-
-1. Follow [Configure access to the $[prodname] web console](../../operations/cnx/access-the-manager.mdx) page to configure the $[prodname] web console access.
+### Set up your Kubernetes cluster to work with Accepted Valuesa non-cluster host or VM
+
+1. Apply the `NonClusterHost` custom resource. This resource enables the cluster-side ingestion endpoint to receive logs from non-cluster hosts. Additionally, it provides a dedicated Typha deployment for managing communication between the non-cluster host agent and Typha.
+
+    To ensure a secure connection, non-cluster host agents will establish an mTLS connection via the `typhaEndpoint` endpoint. As a result, the external ingress or load balancer must be configured to pass through TCP Layer 4 traffic.
+
+    :::note
+
+    Starting from Calico Enterprise version v3.21.0 Early Preview 2, the `typhaEndpoint` field is required for non-cluster host agents to function correctly. Ensure this filed is properly configured to maintain seamless connectivity.
+
+    :::
```

Comment:
>is required for non-cluster host agents to function correctly.

This qualifies the 'required' idea to make it sound like you may lose only some functionality if you don't specify this field. Is that what you mean? 

If not, better to leave it at just 'required'. We don't need to know why.

Reply:
I think we can do without this note. Just make clear in this step that we're setting these two fields and that they're required.

Reply:
Sure. This note is removed.